### PR TITLE
Adding new topic UWP

### PR DESCRIPTION
**Pull request by kolappannathan**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection

---------------------------------------------------------------------

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> UWP is a popular topic with more than a thousand repos today. 

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/uwp/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+aliases: universal-windows-platform
+created_by: Microsoft
+display_name: UWP
```
Comment (talsafran) [inline topics/uwp/index.md]:
```suggestion
display_name: Universal Windows Platform
```

File: topics/uwp/index.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+short_description: UWP is an API that allows a developer to create apps that can run on Windows 10, Windows 10 Mobile, Xbox One and HoloLens.
```
Comment (talsafran) [inline topics/uwp/index.md]: Would maybe 

```suggestion
short_description: An API for creating apps that run on Windows 10, Windows 10 Mobile, Xbox One and HoloLens.
```

File: topics/uwp/index.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+topic: uwp
+wikipedia_url: https://en.wikipedia.org/wiki/Universal_Windows_Platform
+---
+Universal Windows Platform(UWP) is an API that allows a developer to create apps that can run on Windows 10, Windows 10 Mobile, Xbox One and HoloLens without having to create induvidual apps for each platform.
```
Comment (talsafran) [inline topics/uwp/index.md]:
```suggestion
Universal Windows Platform (UWP) is an API that allows developers to create cross-platform applications that are able to run on Windows 10, Windows 10 Mobile, Xbox One and HoloLens.
```

**Comment by kolappannathan** [comment]
@talsafran Made the requested changes

**Comment by talsafran** [comment]
Thank you @kolappannathan 🎉

**Pull request merged**